frontend/packages/operator-lifecycle-manager/src/components/operator-hub/operator-hub-subscribe.tsx

```diff
@@ -248,6 +248,8 @@ export const OperatorHubSubscribeForm: React.FC<OperatorHubSubscribeFormProps> =
   const subscriptionExists = (ns: string) =>
     installedFor(props.subscription.data)(props.operatorGroup.data)(
       props.packageManifest.data[0].status.packageName,
+    )(props.packageManifest.data[0].status.packageCatalogSource)(
+      props.packageManifest.data[0].status.packageNamespace,
```

Comment:
This won't be relevant if we update the function signature as previously suggested, but these are not the correct properties on the PackageManifest API. Should be:
```suggestion
    )(props.packageManifest.data[0].status.catalogSource)(
      props.packageManifest.data[0].status.catalogSourceNamespace,
```